plugins/query-insights/src/main/java/org/opensearch/plugin/insights/core/listener/QueryInsightsListener.java

```diff
@@ -138,6 +139,7 @@ public void onRequestEnd(final SearchPhaseContext context, final SearchRequestCo
             attributes.put(Attribute.TOTAL_SHARDS, context.getNumShards());
             attributes.put(Attribute.INDICES, request.indices());
             attributes.put(Attribute.PHASE_LATENCY_MAP, searchRequestContext.phaseTookMap());
+            attributes.put(Attribute.USER_NAME, request.source().labels().get(DefaultUserInfoLabelingRule.USER_NAME));
```

Comment:
Do we need to add `CUSTOMIZED_TAG` here too?

Reply:
Will move all them into the one single tag.